source/type/data.cpp

```diff
@@ -179,6 +179,10 @@ void Data::build_raptor(size_t cache_size) {
     LOG4CPLUS_DEBUG(logger, "Finished to build data Raptor");
 }
 
+void Data::init_georef_after_loading() {
+    geo_ref->init_inversed_nb_vertex_by_mode();
```

Comment:
Why not call it in the end of `georef::load()`?

Reply:
B R I L L I A N T ! !